src/components/tour/tour_step.tsx

```diff
@@ -131,6 +149,26 @@ export const EuiTourStep: FunctionComponent<EuiTourStepProps> = ({
     );
   }
 
+  const [willMount, setWillMount] = useState<boolean>(false);
+  const animationFrameId = useRef<number>();
+  const node = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (anchor) {
+      animationFrameId.current = window.requestAnimationFrame(() => {
+        node.current = findElement(anchor);
+        if (node) {
```

Comment:
This should be
```suggestion
        node.current = findElement(anchor);
        if (node.current) {
```
I believe, `node` will always be truthy because it's aref object containing

Reply:
🤦‍♂️